backend/analytics/views/collect.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Dict
+
+from ninja import Router, Schema
+
+from backend.utils import SERVER_TIMESTAMP, format_json, get_db
+
+SHOULD_SAVE = False
+
+router = Router()
+
+
+class AnalyticsEvent(Schema):
+    event_type: str
+    data: Dict[str, Any]
+
+
+@router.post("/")
+def post_analytics_data(request, event: AnalyticsEvent):
+    # Add timestamp to event data
+    record = event.dict()
+    record["timestamp"] = SERVER_TIMESTAMP
+
+    # Save event to database
+    db = get_db()
+    ref = db.reference("analytics")
+    # TODO: Remove this check to actually saved the event
+    if SHOULD_SAVE:
```

Comment:
Lauren will remove this check, as well as line 7, to start saving analytics events. We can also print the values to look at what they contain.